### ISSUE TITLE
 Using AGHMN model for Korean dataset

### ISSUE THREAD

**Issue by so-hyeun** [wxjiao/AGHMN#4]
Hello, I want to use this model to do emotion recognition in conversation on the Korean dataset.

If I use word embeddings pre-trained in Korean, can I apply the Korean dataset to this model?

**Comment by wxjiao** [comment]
> Hello, I want to use this model to do emotion recognition in conversation on the Korean dataset.
> 
> If I use word embeddings pre-trained in Korean, can I apply the Korean dataset to this model?

Hi! Yes, the model can be applied to any dataset. But, you may write the function by yourself for assigning the pre-trained embeddings to your vocabulary as the format of your embeddings may be different from that of Word2Vec and GloVe. Actually, the formats of Word2Vec and GloVe are different such that we have different functions for loading them. Just modify some part of the functions and it should work. Thanks!

**Comment by so-hyeun** [comment]
Thanks for the quick reply.

**Comment by so-hyeun** [comment]
I have a question regarding parameter. 
1) In the case of 'args.report_loss', is it correct for the number of train conversations?
2) Is there a special reason to take report_loss/args.report_loss in 'EmoTrain.py'?

3) In addition, I am wondering if there is any special reason for learning without using batch.
Thank you.

**Comment by so-hyeun** [comment]
I have one question.
Currently, batch_size is set to 1 in default. Is it possible to train with increased batch_size? 
Since batch_size is defined in' EmoMain.py', it doesn't seem to be used in other parts of the code.

Thank you.

**Comment by wxjiao** [comment]
> I have one question.
> Currently, batch_size is set to 1 in default. Is it possible to train with increased batch_size?
> Since batch_size is defined in' EmoMain.py', it doesn't seem to be used in other parts of the code.
> 
> Thank you.

Hi. The batch_size is 1 for conversation but with multiple utterances and the corresponding labels.  But, you can use a larger batch by accumulating the gradient for "batch_size" conversations before updating the weights.
